.gitignore

```diff
@@ -78,6 +78,7 @@ web_modules/
 .env.test.local
 .env.production.local
 .env.local
+.env.test
```

Comment:
It would be a good idea to check how many of the files listed in here are actually present in your project. If they are not there (such as the coverage folder, `logs`, `*.logs` etc), they can be removed from this file. If youre not sure, look in your IDE for the file/folder in question and if it isnt there, it can be removed from here